### ISSUE TITLE
Image renderer plugin: *rendering.RenderingService run error: Unrecognized remote plugin message

### ISSUE THREAD

**Issue by simPod** [grafana/grafana#52739]
**What happened**:

I've seen similar issues here in the past like https://github.com/grafana/grafana/issues/40187 and I believe the error is on Grafana's side, not in plugin.

Grafana gives when trying to run:

```
grafana_1               | logger=server t=2022-07-25T15:17:35.361227711Z level=error msg="Stopped background service" service=*rendering.RenderingService reason="Unrecognized remote plugin message: \n\nThis usually means that the plugin is either invalid or simply\nneeds to be recompiled to support the latest protocol."
grafana_1               | logger=tracing t=2022-07-25T15:17:35.361282531Z level=info msg="Closing tracing"
grafana_1               | logger=plugin.grafana-image-renderer t=2022-07-25T15:17:35.36157732Z level=warn msg="plugin failed to exit gracefully"
grafana_1               | logger=http.server t=2022-07-25T15:17:35.36169735Z level=info msg="HTTP Server Listen" address=[::]:3000 protocol=http subUrl=/grafana socket=
grafana_1               | logger=auth t=2022-07-25T15:17:35.362766008Z level=error msg="failed to lock and execute cleanup of expired auth token" error="context canceled"
grafana_1               | logger=provisioning.dashboard t=2022-07-25T15:17:35.363881305Z level=warn msg="Failed to delete orphaned provisioned dashboards" err="context canceled"
grafana_1               | logger=serviceaccounts.store t=2022-07-25T15:17:35.364125524Z level=warn msg="Failed to get usage metrics" error="context canceled"
grafana_1               | logger=infra.usagestats t=2022-07-25T15:17:35.36609348Z level=error msg="Failed to get last sent time" error="context canceled"
grafana_1               | logger=ngalert t=2022-07-25T15:17:35.36613707Z level=error msg="unable to fetch orgIds" msg="context canceled"
grafana_1               | logger=ngalert.multiorg.alertmanager t=2022-07-25T15:17:35.36620195Z level=info msg="starting MultiOrg Alertmanager"
grafana_1               | logger=ngalert t=2022-07-25T15:17:35.3662273Z level=error msg="unable to fetch orgIds" msg="context canceled"
grafana_1               | logger=ticker t=2022-07-25T15:17:35.3662381Z level=info msg=stopped last_tick=2022-07-25T15:17:30Z
grafana_1               | logger=infra.usagestats.collector t=2022-07-25T15:17:35.368273196Z level=error msg="Failed to get system stats" error="context canceled"
grafana_1               | logger=server t=2022-07-25T15:17:35.537548486Z level=error msg="Server shutdown" error="*rendering.RenderingService run error: Unrecognized remote plugin message: \n\nThis usually means that the plugin is either invalid or simply\nneeds to be recompiled to support the latest protocol."
grafana_1               | *rendering.RenderingService run error: Unrecognized remote plugin message:
grafana_1               |
grafana_1               | This usually means that the plugin is either invalid or simply
grafana_1               | needs to be recompiled to support the latest protocol.
```

**What you expected to happen**:

Work

**How to reproduce it (as minimally and precisely as possible)**:

https://github.com/grafana/grafana-image-renderer/issues/358

**Anything else we need to know?**:

**Environment**:
- Grafana version: latest master https://github.com/grafana/grafana/commit/77421907b77d92bcb5d43b42304c218da952cbb2
- OS Grafana is installed on: https://github.com/grafana/grafana/blob/main/packaging/docker/Dockerfile
- User OS & Browser:
- Grafana plugins:
- Others:

**Comment by fcecagno** [comment]
I had this issue while upgrading a deployment in Kubernetes, fixed removing grafana-image-renderer using grafana-cli and then reinstalled it ([link](https://tothecloud.dev/uninstall-grafana-plugin-kubernetes/)).

**Comment by simPod** [comment]
Got fixed with 9.0.7 https://github.com/grafana/grafana-image-renderer/issues/358#issuecomment-1210906923 

Unfortunately, we don't know what fixed it and how to prevent it in the future from happening.
